### DISCUSSION TITLE
Archiving timer/condition ?

### DISCUSSION THREAD

**Discussion by aphelyon-net** [ericaltendorf/plotman#545] [Q&A]
I've set up the archiving configuration, and tested successfully from the guides in the Wiki.
When running plotmanager, my dst. drives are adding new plots, but I don't see archive starting up the rsync process to move them to archive storage.

Is there a condition that needs to be met before this begins archiving plots?
Forgive me if this is a very basic process, but I don't see where this is described in more detail whether the scheduling is dependant on free spac remaining on the dst drive?

**Comment by aphelyon-net** [comment]
I also ran the rsync manually to move one plot. But noticed that this renamed the file on the archive drive to 'plot', and permissions were set to 644? Have I done something wrong?

`rsync --bwlimit=300000 --remove-source-files -P /mnt/stager/plot-k32-2021-05-17-18-44-166a347501aefc4208af48a435763f5d9cf452f168cb1c7dfac6561be02213d2.plot rsync://dan@10.1.0.126:12000/chia/plots
plot-k32-2021-05-17-18-44-166a347501aefc4208af48a435763f5d9cf452f168cb1c7dfac6561be02213d2.plot
108,846,221,861 100%  158.70MB/s    0:10:54 (xfr#1, to-chk=0/1)`

Archive host:
`-rwxrwxrwx 1 dan  dan  102G May 17 04:11 plot-k32-2021-05-16-20-16-0f8f009c46a0aa00b524b1897254a1d4a71e7218b405a831d62f10c8bc357756.plot
-rwxrwxrwx 1 dan  dan  102G May 17 15:55 plot-k32-2021-05-17-10-08-ca0f08d0d927483916e534841b0699d3ccee1ce3744764407a97f581547904d4.plot
-rwxrwxrwx 1 dan  dan  102G May 17 16:46 plot-k32-2021-05-17-10-38-612df6ccc3819c8c6b710d3e0c0d59b12f8403db1c4785d1177c9eb3624b978e.plot
-rw-r--r-- 1 dan  dan  102G May 18 01:59 plots`

Have managed to catch the archive rsync process working.. however it appears it's renaming the file. Following the wiki configuration everything looks correct? 

![Screen Shot 2021-05-18 at 10 21 47 am](https://user-images.githubusercontent.com/69456746/118581237-ded5ee80-b7c3-11eb-9b63-149cd78b49a6.png)

![Screen Shot 2021-05-18 at 10 21 59 am](https://user-images.githubusercontent.com/69456746/118581245-e39aa280-b7c3-11eb-9221-2c44a3db1a99.png)

Has renamed the file to the archive directory name:
![Screen Shot 2021-05-18 at 10 34 24 am](https://user-images.githubusercontent.com/69456746/118581728-b1d60b80-b7c4-11eb-8b5a-ef04c43bfaf4.png)


My archive directories.
![Screen Shot 2021-05-18 at 11 30 10 am](https://user-images.githubusercontent.com/69456746/118586133-c7e7ca00-b7cc-11eb-8d3b-d9ea7fab85e3.png)

My current plotman configruation for archiving. and archive host rsync module configuration:
![Screen Shot 2021-05-18 at 11 31 40 am](https://user-images.githubusercontent.com/69456746/118586099-b7cfea80-b7cc-11eb-8d3c-8740855634c0.png)
![Screen Shot 2021-05-18 at 11 31 40 am](https://user-images.githubusercontent.com/69456746/118586112-bc949e80-b7cc-11eb-8a39-9c462c9c01f4.png)

**Reply by aphelyon-net** [reply]
Very strange. Its now started working correctly. I had some rsync processes that looked to be stuck; so killed those off.. and then after a short time it started to work correctly.

**Comment by cyperbg** [comment]
At what interval does it move files?
As soon as they are ready?

**Reply by aphelyon-net** [reply]
Yes, after monitoring further. As soon as a .plot file exists on the dst drive, rsync initiates to copy the file over.
I think the issue I might have had, was that zombie rsync processes existed that were causing issues.
Everything seems to be working now.

Will monitor further and see if any other strange behaviour happens again.

**Reply by cyperbg** [reply]
Do you know how to setup the archiving option to use local drives instead of a remote machine.
